app/templates/compare_revisions.html

```diff
@@ -41,89 +45,96 @@
-            <table>
-              <caption>{{ diff.label }}</caption>
-              <thead>
-                <tr>
-                  <th class="line-number"><span class="visuallyhidden">Revision 1 line number</span></th>
-                  <th class="line-content"><span class="visuallyhidden">Revision 1 changes</span></th>
-                  <th class="line-number"><span class="visuallyhidden">Revision 2 line number</span></th>
-                  <th class="line-content"><span class="visuallyhidden">Revision 2 changes</span></th>
-                </tr>
-              </thead>
-              <tbody>
-                {% for line in diff.revisions %}
-                  <tr class='diff-row'>
-                    {{ line }}
-                  </tr>
-                {% endfor %}
-              </tbody>
-            </table>
+  {%
+    with
+    smaller = true,
+    heading = service['serviceName'],
+    context = service['supplierName']
+  %}
+    {% include "toolkit/page-heading.html" %}
+  {% endwith %}
+  {%
+    with
+      url = "/g-cloud/services/{}".format(service.id),
+      text = "View service"
+  %}
+    {% include "toolkit/secondary-action-link.html" %}
+  {% endwith %}
+    <div class="grid-row">
+      <div class="column-two-thirds page-section">
+        <p>
+          {% if all_update_events is none %}
+            Multiple
+          {% else %}
+            {{ all_update_events|length }}
           {% endif %}
-        {% endfor %}
-      {% endfor %}
-      </div><!-- end of .diff -->
-
-      {% if current_user.has_role('admin') %}
-      <form action="{{ url_for('.update_service_status', service_id=service_data.id ) }}" method="post">
-          <div style="display:none;"><input id="csrf_token" name="csrf_token" type="hidden" value="{{ csrf_token() }}"></div>
-          <fieldset class="question">
-              <legend class="question-heading">
-                  Service status
-              </legend>
-              <label class="selection-button">
-                  <input type="radio" name="service_status" id="service_status_disabled" value="removed" {% if service_data['status'] == 'disabled' %}checked="checked"{% endif %} />
-                  Removed
-              </label>
-              <label class="selection-button">
-                  <input type="radio" name="service_status" id="service_status_private" value="private" {% if service_data['status'] == 'enabled' %}checked="checked"{% endif %} />
-                  Private
-              </label>
-              {% if service_data.status != 'disabled' %}
-              <label class="selection-button">
-                  <input type="radio" name="service_status" id="service_status_published" value="public" {% if service_data['status'] == 'published' %}checked="checked"{% endif %} />
-                  Public
-              </label>
+          unapproved {{ pluralize((all_update_events or (oldest_update_events+latest_update_events))|length, "edit", "edits") }}
```

Comment:
Looking at the commit message in utils it apparently "works in a similar way to Rails's pluralize function", which might explain the perceived weirdness :smile: